src/main/java/com/auth0/json/mgmt/Scope.java

```diff
@@ -0,0 +1,43 @@
+package com.auth0.json.mgmt;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Scope {
+    @JsonProperty("description")
+    private String description;
+    @JsonProperty("value")
+    private String value;
+
+    @JsonCreator
+    public Scope(@JsonProperty("value") String value,
+                 @JsonProperty("description") String description) {
+        this.description = description;
+        this.value = value;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Remove this method as we chose not to provide it on the other entities.

Reply:
done